lib/editing/editor.dart

```diff
@@ -121,5 +121,7 @@ class Completion {
 
   final String type;
 
+  bool get isMethodWithArguments => displayString.contains("(") && !displayString.contains("()");
```

Comment:
I'd rather that we read this by looking at the length of the parameterNames list, rather than heuristics based on the display string. These are a bit fragile and may break in unexpected ways if we change the way the test is displayed.